userspace/libsinsp/parsers.cpp

```diff
@@ -666,8 +669,11 @@ bool sinsp_parser::reset(sinsp_evt *evt) {
 			//
 			int fd_location = get_fd_location(etype);
 			ASSERT(evt->get_param_info(fd_location)->type == PT_FD);
-			evt->get_tinfo()->m_lastevent_fd = evt->get_param(fd_location)->as<int64_t>();
-			evt->set_fd_info(evt->get_tinfo()->get_fd(evt->get_tinfo()->m_lastevent_fd));
+			if((etype != PPME_SOCKET_SENDMMSG_E && etype != PPME_SOCKET_RECVMMSG_E) ||
```

Comment:
I think that if we remove `fd` param from `PPME_SOCKET_RECVMMSG_E` (and remove the `EF_USES_FD` flag too of course) we can skip this check.